### ISSUE TITLE
Catalog "Add" creates a white on white edit box so you cannot see what you are typing... 

### ISSUE THREAD

**Issue by RonnyTNL** [samsonwang/LaunchyQt#95]
**Describe the bug**
Catalog "Add" creates a white on white edit box so you cannot see what you are typing... (Introduced after QT upgrade I think)

**To Reproduce**
1. Press ALT-SPACE
2. Right mouse -> Options
3. Click on Catalog TAB
4. Click + to add a new folder
5. A new item is added to the list, but the text is White on White which makes it impossible to see what you are typing.

**Expected behavior**
Preferably a box and black text on white background.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - Operation System: [Win10x64 22H2]
 - Launchy Version: [e.g. 3.1.5]

**Additional context**
N/A

**Comment by Stout001** [comment]
Yes. Quite annoying. As it is now I only have Windows start menu and when typing for an app I only get suggestions to the web.

**Comment by samsonwang** [comment]
fixed in v3.1.7
